var/spack/repos/builtin/packages/babelstream/package.py

```diff
@@ -5,100 +5,217 @@
 from spack.package import *
 
 
-def find_model_flag(str):
-    res = re.findall(r"\+(\w+)", str)
-    if not res:
-        return ""
-    return res
-
-
-class Babelstream(CMakePackage, CudaPackage, ROCmPackage):
+class Babelstream(CMakePackage, CudaPackage, ROCmPackage, MakefilePackage):
     """Measure memory transfer rates to/from global device memory on GPUs.
     This benchmark is similar in spirit, and based on, the STREAM benchmark for CPUs."""
 
     homepage = "https://github.com/UoB-HPC/BabelStream"
-    url = "https://github.com/UoB-HPC/BabelStream/archive/refs/tags/v4.0.tar.gz"
+    url = "https://github.com/UoB-HPC/BabelStream/archive/refs/tags/v5.0.tar.gz"
     git = "https://github.com/UoB-HPC/BabelStream.git"
+    version("5.0", sha256="1a418203fbfd95595bdc66047e2e39d8f1bba95a49725c9ecb907caf1af2521f")
     version("4.0", sha256="a9cd39277fb15d977d468435eb9b894f79f468233f0131509aa540ffda4f5953")
+    version("3.4", sha256="e34ee9d5ccdead019e3ea478333bcb7886117d600e5da8579a626f6ee34209cf")
+    version("3.3", sha256="4c89c805b277d52776feeb7a8eef7985a0d9295ce3e0bb2333bf715f724723cf")
+    version("3.2", sha256="20309b27ddd09ea37406bcc6f46fd32e9372bf3d145757e55938d19d69cdc49d")
+    version("3.1", sha256="be69e6085e8966e12aa2df897eea6254b172e5adfa03de0adbb89bc3065f4fbe")
+    version("3.0", sha256="776219c72e0fdc36f134e6975b68c7ab25f38206f8f8af84a6f9630648c24800")
+    version("1.0", sha256="3cfb9e45601f1f249878355c72baa6e6a61f6c811f8716d60b83c7fb544e1d5c")
     version("main", branch="main")
-    version("develop", branch="develop")
-
-    depends_on("cxx", type="build")  # generated
-
-    maintainers("tomdeakin", "kaanolgu", "tom91136", "robj0nes")
-
+    maintainers("tomdeakin", "kaanolgu", "tom91136")
+    # Previous maintainers: "robj0nes"
+    depends_on("cxx", type="build")
     # Languages
-    # Also supported variants are cuda and rocm (for HIP)
-    variant("sycl", default=False, description="Enable SYCL support")
-    variant("sycl2020", default=False, description="Enable SYCL support")
-    variant("omp", default=False, description="Enable OpenMP support")
-    variant("ocl", default=False, description="Enable OpenCL support")
-    variant("tbb", default=False, description="Enable TBB support")
-    variant("acc", default=False, description="Enable OpenACC support")
-    variant("thrust", default=False, description="Enable THRUST support")
-    variant("raja", default=False, description="Enable RAJA support")
-    variant("stddata", default=False, description="Enable STD-data support")
-    variant("stdindices", default=False, description="Enable STD-indices support")
-    variant("stdranges", default=False, description="Enable STD-ranges support")
+    # in the future it could be possible to add other languages too
+    variant(
+        "languages",
+        default="cxx",
+        values=("cxx", "fortran"),
+        description="Languages Babelstream Spack Package Support"
+        )
```

Comment:
This implies that it's one or the other, but never both. Is that correct?

Reply:
Yes